lib/index.js

```diff
@@ -73,18 +73,19 @@ module.exports = class DatadogTransport extends Transport {
     const queryString = querystring.encode(query)
     const api = querystring ? `${this.api}?${queryString}` : this.api
 
-    try {
-      // Perform the writing to the remote service
-      await fetch(api, {
-        method: 'POST',
-        headers: {
-          'content-type': 'application/json'
-        },
-        body: JSON.stringify(logs)
-      })
-    } catch (err) {
-    } finally {
+    // Perform the writing to the remote service
+    fetch(api, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json'
+      },
+      body: JSON.stringify(logs)
+    }).then(() => {
+      this.emit('logged', info)
```

Comment:
These new events were influenced by https://github.com/adrianhall/winston-splunk-httplogger/blob/main/index.js#L148-L151, https://github.com/TheAppleFreak/winston-slack-webhook-transport/blob/master/slackHook.js#L45-L52 and https://github.com/winstonjs/winston/blob/master/lib/winston/transports/http.js#L57-L61. They also feel useful for supporting https://github.com/itsfadnis/datadog-winston/pull/21